### PR TITLE
Game logic

### PR DESCRIPTION
**Pull request by sergiocortessat**
Milestone 3. We have added all logic to the game. It comprehends modules. the main.rb is in charge of playing the game while game.rb is in charge of the overall logic. the game is currently  able to receive 2 players, symbols, winners, draws, and user inputs

### CODE REVIEW

**Comment by mimipeshy** [comment]
### STATUS: CHANGES REQUIRED 🔴 ♻️ 

Hey there @sergiocortessat and @arikarim 👋🏾 👋🏾 it is good to make it this far, your project is really nice, congratulations on the good work you have done so far. Your project works well, you have validated inputs as well. However, a few changes need to be made before the project can be marked as complete

- [ ] All game-logic related resources (classes and/or modules) are kept in the lib directory. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/lib_dir_exist.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))
- [ ] When all logic is moved to `lib` ensure tha there are no `puts/print` and `gets.chomp` outside `main.rb` file.
- [x] After a game is complete and the user enters `ok` to continue playing a new game, let me paint the scenario
- A user has pressed `ok`
- A tictac toe welcome menu pops
- player 1 enters their name
- player 1 chooses a symbol
- player 2 enters a name
- A board is displayed showing the last game's results
- The user is again asked to press `ok` to play
- the user is taken back to step 1
Kindly fix this

That's all :smile: kindly submit another review when done.

_Feel free to reach out to me on slack if you have any questions or something is unclear, or just comment on this PR and tag my name and I will be glad to help_
Regards
Peris Ndanu

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🔴 ♻️
Hey there @sergiocortessat and @arikarim 👋🏾 👋🏾 it is good to make it this far, your project is really nice, congratulations on the good work you have done so far. Your project works well, you have validated inputs as well. However, a few changes need to be made before the project can be marked as complete
- [x] All game-logic-related resources (classes and/or modules) are kept in the lib directory. 
- [x] Two players should not have the same name. 
- [x] There is no draw condition is present. Please add it. 
hat's all 😄 kindly submit another review when done.

Feel free to reach out to me on slack if you have any questions or something is unclear, or just comment on this PR and tag my name and I will be glad to help
Regards
Arslan Bisharat

**Comment by arikarim** [comment]
thanks very much for the review t was really helpful, we already look at it and did all the changes requested,.
please have a look again.

**Review by kapkev — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻
### Well done on making the previously required changes, your project is almost done, just a few things before we move on:

- [x] Looks like your project still has puts outside the `main.rb` file, please keep all the puts inside the main.rb file.

After making the required changes, please submit it again.
<hr>
If you have any question, feel free to reach me through slack

**Comment by arikarim** [comment]
Moved all the puts outside main file

File: lib/board.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    @cells = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+  end
+
+  def show
+    puts "#{cells[0]} | #{cells[1]} | #{cells[2]}"
+    puts '---+---+---'
+    puts "#{cells[3]} | #{cells[4]} | #{cells[5]}"
+    puts '---+---+---'
+    puts "#{cells[6]} | #{cells[7]} | #{cells[8]}"
+  end
```
Comment (acushlakoncept) [inline lib/board.rb]: No `puts/print` and `gets.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

Kindly move it to `main.rb`

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

### Hello @sergiocortessat and @arikarim, impressive work thus far.
Let's make a few changes and we'll be done.

- [x] ### Looks like you missed out on the `puts` statement outside `main.rb` kindly see the comments below
- [x] The screenshot on your readme file seems broken, no image is loading on my end. kindly fix this.
- [x] In your readme file, kindly update the install section of the readme file to reflect the current project - You have "Download a copy of the enumerables.rb file".

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Review by lekegitrepo — APPROVED** [review]
## Status: Approved :heavy_check_mark:
### Hi @sergiocortessat and @arikarim  :100: :1st_place_medal:
#### You have done a good job by implementing the required changes. Please, try to update the readme in the game instruction milestone. This is approved :white_check_mark:

#### _Reviewed by Temitayo Adeleke_

**Pull request merged**